### DISCUSSION TITLE
Chunk uploads to file library

### DISCUSSION THREAD

**Discussion by mattbab** [directus/directus#12937] [Feature Requests]
I have a Directus instance which I use for file hosting. This instance runs on Azure and has SSL terminated by Cloudflare. Cloudflare has a 100MB max POST payload size for free and Pro accounts. When uploading a file to the file library that is over 100MB, Cloudflare responds with a 413 error. As I understand, this wouldn't occur if the Directus web UI used chunked file uploads. Doing so would also help in other limited hosting scenarios.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
